src/components/custom/modal.tsx

```diff
@@ -7,14 +7,21 @@ import {
   DialogTitle,
   DialogTrigger,
 } from "@/components/ui/dialog";
+import { cn } from "@/lib/utils";
 
 type Props = {
   modalTitle?: string;
   buttonTitle?: string;
   children: React.ReactNode;
+  className?: string;
 };
 
-export const Modal = ({ modalTitle, buttonTitle, children }: Props) => {
+export const Modal = ({
+  modalTitle,
+  buttonTitle,
+  children,
+  className,
+}: Props) => {
```

Comment:
Props => ModalProps

Reply:
I give own type names only for types that I use elsewhere in the project.

Reply:
Ok